README.md

```diff
@@ -4,29 +4,28 @@
 
 # Checkout V2
 
-This action checks-out your repository under `$GITHUB_WORKSPACE`, so your workflow can access it.
+This action checks-out your repository under `$GITHUB_WORKSPACE`.
 
-By default, the repository that triggered the workflow is checked-out, for the ref/SHA that triggered the event.
+Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set `fetch-depth` to fetch more history. Refer [here](https://help.github.com/en/articles/events-that-trigger-workflows) to learn which commit `$GITHUB_SHA` points to for different events.
 
-Refer [here](https://help.github.com/en/articles/events-that-trigger-workflows) to learn which commit `$GITHUB_SHA` points to for different events.
+The auth token is persisted in the local git config. This enables your scripts to run authenticated git commands. The token is removed during post-job cleanup. Use the input `persist-credentials` to opt-out.
+
+When Git 2.18 or higher is not in your PATH, falls back to the REST API to download the files.
 
 # What's new
 
-- Improved fetch performance
-  - The default behavior now fetches only the commit being checked-out
+- Improved performance
```

Comment:
I simplified the `What's new` section to make it easier to skim

Also the summary is more verbose now, so I didn't want to repeat all details.